monkey/monkey_island/cc/services/initialize.py

```diff
@@ -11,12 +11,12 @@
 
 def initialize_services(data_dir: Path) -> DIContainer:
     container = DIContainer()
+    container.register_instance(AWSInstance, AWSInstance())
+
     container.register_instance(
         IFileStorageService, DirectoryFileStorageService(data_dir / "custom_pbas")
     )
-
-    # Takes a while so it's best to start it in the background
-    Thread(target=aws_service.initialize, name="AwsService initialization", daemon=True).start()
+    container.register_instance(AWSService, container.resolve(AWSService))
```

Comment:
Hmm, so this is linked to line 14, right? You can't swap these lines

Reply:
Right